prow/cmd/autobump/autobump.sh

```diff
@@ -128,9 +128,24 @@ create-gerrit-pr() {
   git config http.cookiefile "${creds}"
   git remote add upstream "${GERRIT_HOST_REPO}"
 
-	change_id="$(get-change-id)"
+	local change_id="$(get-change-id)"
 	echo "Commit will have Change-Id: ${change_id}"
 
+	# Check for an existing open PR and see if it needs to be updated.
+	echo "Checking for an existing open Gerrit PR to update..."
+	git fetch upstream "+refs/changes/*:refs/remotes/upstream/changes/*" 2> /dev/null
+	local pr_commit="$(git log --all --grep="Change-Id: ${change_id}" -1 --format="%H")"
+	if [[ -n "${pr_commit}" ]]; then
+		local pr_version="$(git show "${pr_commit}:${PLANK_DEPLOYMENT_FILE}" | extract-version)"
```

Comment:
Wouldn't `git diff $pr_commit` be better than just checking the plank version? AKA if any of the files change we want to push a new commit.

Reply:
Yes thats definitely better. Fixed.